ocs_ci/ocs/utils.py

```diff
@@ -925,45 +956,115 @@ def run_must_gather(log_dir_path, image, command=None, cluster_config=None):
             cluster_config=cluster_config,
         )
     except CommandFailed as ex:
-        log.error(get_helper_pods_output())
         log.error(
             f"Failed during must gather logs! Error: {ex}"
             f"Must-Gather Output: {mg_output}"
         )
+        export_mg_pods_logs(log_dir_path=log_dir_path, mg_output=mg_output)
+
     except TimeoutExpired as ex:
-        log.error(get_helper_pods_output())
         log.error(
-            f"Timeout {must_gather_timeout}s for must-gather reached, command"
-            f" exited with error: {ex}"
+            f"Failed during must gather logs! Error: {ex}"
             f"Must-Gather Output: {mg_output}"
         )
+        export_mg_pods_logs(log_dir_path=log_dir_path, mg_output=mg_output)
     return mg_output
 
 
-def get_helper_pods_output():
+def export_mg_pods_logs(log_dir_path, mg_output=None):
     """
-    Get the output of "oc describe mg-helper pods"
+    Export must gather pods logs
+
+    Args:
+        mg_output (str): the output of "oc adm must-gather --image=quay.io/rhceph-dev/ocs-must-gather:tag" command
+        log_dir_path (str): the path of copying the logs
```

Comment:
please sort the args here the same way as they are sorted in the function name

Reply:
I deleted `mg_output` arg